js/utils.js

```diff
@@ -886,36 +890,48 @@ Fliplet.Registry.set('dynamicListUtils', (function() {
           return data.record;
         }
 
-        var image = _.get(data, ['record', 'data', data.field.column]);
+        if (data.field.from === 'summary') {
```

Comment:
The current implementation doesn't rely on any checks of `data.field.from`. You have now added checks against `"summary"` and `"detail"`. Is there any possibility where the `data.field.from` could be neither `"summary"` or `"details"`? An empty/null value perhaps?

Is it safer to just go with `if (data.field.from === 'detail') { /* ... */ } else { /* ... */ }`?

Reply:
@tonytlwu As far as I tested I didn't see any other option except `summary` or `detail`.
Therefore I'll go with this `if (data.field.from === 'detail') { /* ... */ } else { /* ... */ }`.

Reply:
> Therefore I'll go with this `if (data.field.from === 'detail') { /* ... */ } else { /* ... */ }`.

@YaroslavOvdii You say this, but you haven't committed any code to apply the changes.

Reply:
@tonytlwu remade if statement.